packages/website/docs/autocomplete-js.md

```diff
@@ -141,6 +141,60 @@ type ClassNames = Partial<{
 }>;
 ```
 
+### `components`
+
+Components to register to the Autocomplete rendering lifecycles. They are then available in the [`templates`](templates), in [`render`](#render) and in [`renderNoResults`](#rendernoresults).
+
+```jsx
+import { render } from 'preact';
+import { SearchByAlgolia } from './my-components';
+
+autocomplete({
+  // ...
+  components: {
+    SearchByAlgolia,
+  },
+  render({ sections, components }, root) {
+    render(
+      <Fragment>
+        <div className="aa-PanelLayout aa-Panel--scollable">{sections}</div>
+        <components.SearchByAlgolia />
```

Comment:
This can be confusing because we ship a `SearchByAlgolia` component in the Layout API.

```suggestion
import { MyComponent } from './my-components';

autocomplete({
  // ...
  components: {
    MyComponent,
  },
  render({ sections, components }, root) {
    render(
      <Fragment>
        <div className="aa-PanelLayout aa-Panel--scollable">{sections}</div>
        <components.MyComponent />
```